bluebutton-data-pipeline-fhir-load/src/test/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformerTest.java

```diff
@@ -639,40 +688,119 @@ public void transformInsertPartDEvent() throws FHIRException {
 		Assert.assertNotNull(pdeBundleWrapper.getResult());
 
 		Bundle pdeBundle = pdeBundleWrapper.getResult();
-		Assert.assertEquals(1, pdeBundle.getEntry().size());
+		Assert.assertEquals(5, pdeBundle.getEntry().size());
```

Comment:
Just for maintenance/sanity, I've started adding a comment to these bundle size checks that lists the resources expected to be in it. I'd recommend you do the same.